test/micro/org/openjdk/bench/java/lang/foreign/LoopOverPollutedSegments.java

```diff
@@ -64,7 +64,9 @@ public class LoopOverPollutedSegments {
     byte[] arr;
     long addr;
 
-    static final VarHandle intHandle = JAVA_INT.arrayElementVarHandle();
+    static final ValueLayout.OfInt JAVA_INT_UNALIGNED = JAVA_INT.withBitAlignment(8);
+    static final ValueLayout.OfFloat JAVA_FLOAT_UNALIGNED = JAVA_FLOAT.withBitAlignment(8);
+    static final VarHandle intHandleUnaligned = JAVA_INT_UNALIGNED.arrayElementVarHandle();
```

Comment:
To match `LoopOverNonConstantHeap`, this should probably be named `VH_INT_UNALIGNED`.

--------------------------------------------------------------------------------

Maybe these could also be moved into `org.openjdk.bench.java.lang.foreign.Utils`[^1]

[^1]: https://github.com/openjdk/jdk/blob/7b774297b1d04e104a988ea5bd2f2b04c8de3461/test/micro/org/openjdk/bench/java/lang/foreign/Utils.java#L29-L43

Reply:
I noted other possible cleanup for benchmarks, I'll work on something after we integrate this PR as I'd like to minimize the churn.